src/App.jsx

```diff
@@ -1,71 +1,102 @@
 import React, { useEffect } from 'react';
 import { BrowserRouter, Switch, Route } from 'react-router-dom';
-import { Campaigns, Campaign, Homepage, AboutUs, NowUApp, GetInTouch, Press, FAQs, err404, Blog, BlogArticle } from './domain';
-import { AnnounceBar, TopNav, Footer, NewsletterSignupModal } from './components';
-import { ModalService } from "./services";
+import {
+    Campaigns,
+    Campaign,
+    Homepage,
+    AboutUs,
+    NowUApp,
+    GetInTouch,
+    Press,
+    FAQs,
+    err404,
+    Blog,
+    BlogArticle,
+} from './domain';
+import {
+    AnnounceBar,
+    Navbar,
+    Footer,
+    NewsletterSignupModal,
+} from './components';
+import { ModalService } from './services';
 import './App.scss';
 
 const App = () => {
-  const openNewsletterSignupModal = () => {
-    const newsletterSignupDismissed = sessionStorage.getItem('newsletter-signup-dismissed');
-    const newsletterSignupCompleted = localStorage.getItem('newsletter-signup-completed');
+    const openNewsletterSignupModal = (show = false) => {
```

Comment:
`show` isn't very clear as a name. Something like `forceShowNewsletterSignup` is better.